src/courses/delta/README.md

```diff
@@ -0,0 +1,22 @@
+---
+order: 01
+next: 02.md
+title: Delta Process Developer Class
+author: Daniel Medina, George Dias
+---
+
+## 1.1 Class Overview
+
+The purpose of this class is to demonstrate how the MITRE SAF Delta tool can be used to facilitate changes to InSpec profile baselines. This process is often utilized to create a new baseline profile from older revisions of the same baseline.
+
+### 1.1.1 Class Objectives
```

Comment:
```suggestion
## 1.2 Class Objectives
```

I recommend only having a certain level of index if there will be more than one. For example try to only make a 1.1 if there will be a 1.2 or a 1.1.1 only if there will be a 1.1.2